Psychtoolbox/PsychHardware/NetStation.m

```diff
@@ -32,6 +32,27 @@
 %           Synchronize to the connected host. "SynchLimit" could specify the maximum allowed difference
 %           IN MILLISECONDS. Default is 2.5 ms.
 %
+%   NetStation( 'getntpsynchronize', ntpserver )
```

Comment:
Please match capitalization of 'getntpsynchronize' with style of other NetStation commands.